src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionInfoPolling.tsx

```diff
@@ -28,11 +45,15 @@ export const ArtworkSidebarAuctionPolling: React.FC<Props> = ({
     },
     intervalTime: 10000,
     key: artwork.internalID,
+    clearWhen: isClosed,
```

Comment:
This was an unrelated update (probably should have been its own commit, but was so minor). 

The `usePoll` hook has a `clearWhen` property which will not perform polling (or stop if it was) when true (in https://github.com/artsy/force/blob/df3c95874980cc6806afa867207790cb1548fe2e/src/v2/Apps/Conversation/Utils/usePoll.ts#L30)

I realized we were making these polling requests when on a lot page where the auction was already closed, which is a lot of extra requests that aren't needed.